src/storage/page_buffer.c

```diff
@@ -2171,6 +2171,16 @@ pgbuf_fix_release (THREAD_ENTRY * thread_p, const VPID * vpid, PAGE_FETCH_MODE f
 	  PGBUF_BCB_CHECK_MUTEX_LEAKS ();
 	  pgbuf_unfix (thread_p, pgptr);
 	  return NULL;
+	case OLD_PAGE_IF_IN_BUFFER_OR_IN_TRANSIT:
```

Comment:
Nit : can these two last cases be moved first ? ( I always expect "default" to be at the end of switch.)

Reply:
I would like that too. But I prefer to keep changes in existing files to a minimum to ease merging from `develop`.